<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="vistaPrograma">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="salidaResultado">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_programa">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Programa</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>1-Inserción</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2-Intercambio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3-Selección</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonEntrada">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de_AppMenu"/>
    <addaction name="actionInforme_de_errores"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionAcerca_de_AppMenu">
   <property name="text">
    <string>Acerca de AppMenu</string>
   </property>
  </action>
  <action name="actionInforme_de_errores">
   <property name="text">
    <string>Informe de errores</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
